misc/testerina/modules/testerina-core/src/test/java/org/ballerinalang/testerina/test/DataDrivenTests.java

```diff
@@ -0,0 +1,83 @@
+/*
+ *
+ *   Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing,
+ *   software distributed under the License is distributed on an
+ *   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *   KIND, either express or implied.  See the License for the
+ *   specific language governing permissions and limitations
+ *   under the License.
+ * /
+ */
+
+package org.ballerinalang.testerina.test;
+
+import org.ballerinalang.testerina.core.BTestRunner;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Tests to cover data set based testing.
+ */
+public class DataDrivenTests {
+
+    Path[] filePaths = { Paths.get("src/test/resources/annotations-test/data-driven-positive-test.bal") };
+    Path[] filePathsGrouped = { Paths.get("src/test/resources/annotations-test/data-driven-with-groups-test.bal") };
```

Comment:
Shall we make these two variables private?

Reply:
Did the changes.